components/HTTP/src/RESTServer.cpp

```diff
@@ -40,8 +42,15 @@ typedef struct {
     char scratch[SCRATCH_BUFSIZE];
 } rest_server_context_t;
 
-static const char* TAG = "RESTServer";
 
+/*
+ * Structure holding server handle
+ * and internal socket fd in order
+ * to use out of request send
+ */
+
+
```

Comment:
This comment can be removed, seems it was for an structure that was there alright?